--- brick.py
+++ brick-new.py
@@ -13,7 +13,7 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 
-from time import sleep
+import time # sleep()
 from threading import Lock
 from nxt.error import FileNotFound, ModuleNotFound
 from nxt.telegram import OPCODES, Telegram
@@ -224,7 +224,7 @@
 
     def play_tone_and_wait(self, frequency, duration):
         self.play_tone(frequency, duration)
-        sleep(duration / 1000.0)
+        time.sleep(duration / 1000.0)
 
     def __del__(self):
         self.sock.close()
